src/scenes/UserGalleryPage/UserGalleryLayout.tsx

```diff
@@ -81,6 +81,7 @@ export const UserGalleryLayout = ({ userRef, queryRef }: Props) => {
       <UserGalleryHeader
         userRef={user}
         showMobileLayoutToggle={showMobileLayoutToggle}
+        showQRCode={isMobile}
```

Comment:
I am showing the QR code anytime the window width is mobile or smaller - let me know if we should change this